### PR TITLE
Added missing cronjob and node atrributes : https://github.com/kubern…

### PR DESCRIPTION
**Pull request by sumitk23**
…etes/kube-state-metrics/issues/1509

Tested by running ksm locally and using unit tests. Screenshots attached for new fields in metrics endpoint.

### CODE REVIEW

**Comment by sumitk23** [comment]
cronjob_ksm
node_ksm

File: internal/store/cronjob_test.go [hunk 1]
```diff
@@ -33,6 +33,8 @@ var (
 	SuspendTrue                      = true
 	SuspendFalse                     = false
 	StartingDeadlineSeconds300 int64 = 300
+	SuccessfulJobHistoryLimit3 int32 = 3
```
Comment (Sumit1991Saha) [inline internal/store/cronjob_test.go:36]: why a value 3 being used, shouldn't we save more successful jobs?

File: internal/store/node.go [hunk 2]
```diff
@@ -90,6 +90,7 @@ func createNodeInfoFamilyGenerator() generator.FamilyGenerator {
 				"kubeproxy_version",
 				"provider_id",
 				"pod_cidr",
+				"system_uuid",
```
Comment (Sumit1991Saha) [inline internal/store/node.go:93]: why this is needed?

File: internal/store/cronjob_test.go (hunk 1 above)
Comment (sumitk23) [reply internal/store/cronjob_test.go:36]: This is just a unit test we can keep any value just have to make sure use the same for verification. Used 3 as it's the default value for this field in Kubernetes if unspecified.

File: internal/store/node.go (hunk 2 above)
Comment (sumitk23) [reply internal/store/node.go:93]: We report this field in Api server mode, hence need this in KSM mode to maintain parity.

**Pull request merged**